### ISSUE TITLE
The topic in the added GridCells plug-in is not displayed

### ISSUE THREAD

**Issue by qisongH** [ros-visualization/rviz#1720]
After adding the GridCells plug-in through the "Add - > By topic" interface in RViz, the topic in the added GridCells plug-in is not displayed, which is different from that of other plug-ins added in the same way.
![image](https://user-images.githubusercontent.com/74614539/156989058-51d25a2a-ea6d-48ba-84c1-7c94b39dbbbb.png)
![image](https://user-images.githubusercontent.com/74614539/156989343-38ccd296-c6b7-4be8-9268-9c58038a2fec.png)

### Your environment
* OS Version: e.g. Ubuntu 18.04
* ROS Distro: [Melodic]
* RViz, Qt, OGRE, OpenGl version as printed by rviz:
```
[ INFO] [1646653665.369882000]: rviz version 1.13.18
[ INFO] [1646653665.369933300]: compiled against Qt version 5.9.5
[ INFO] [1646653665.369944800]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1646653665.373622300]: Forcing OpenGl version 0.
[ INFO] [1646653665.491788300]: Stereo is NOT SUPPORTED
[ INFO] [1646653665.492296300]: OpenGL device: llvmpipe (LLVM 10.0.0, 256 bits)
[ INFO] [1646653665.492382900]: OpenGl version: 3.1 (GLSL 1.4).
```

![image](https://user-images.githubusercontent.com/74614539/156989349-c9c2ce59-bfcc-4b40-adf1-426538604fe0.png)

**Comment by rhaschke** [comment]
You mean, the topic field is empty, while it should show the selected topic name?
Which version of rviz are you using?

**Comment by rhaschke** [comment]
I was able to reproduce the issue on Melodic, but not on Noetic. Can you confirm?

**Comment by qisongH** [comment]
> You mean, the topic field is empty, while it should show the selected topic name? Which version of rviz are you using?
Yes, the topic field should show the selected topic name and display GridCells in 3D area. The version of rviz is 1.13.18.

**Comment by qisongH** [comment]
> I was able to reproduce the issue on Melodic, but not on Noetic. Can you confirm? I won't dig into the code to fix a bug that's only present in an outdated release. Please consider updating or file a fixing PR for Melodic. Thanks.

Sorry, this issue occurred on Melodic, I re-edited the environment in which the issue occurred.

**Comment by rhaschke** [comment]
Fixed via #1722
